features/upgrade/README.md

```diff
@@ -1,178 +1,5 @@
-# Upgrading from rspec-rails-2.x to rspec-rails-3
+# Upgrading from rspec-rails 3.x to version 4
 
-For detailed information on the general RSpec 3.x upgrade process see the [RSpec
-Upgrade docs](https://relishapp.com/rspec/docs/upgrade).
+There is nothing that needs to be done to upgrade from RSpec Rails 3 to version 4.0.
```

Comment:
Can't think of anything, really.
Should we mention that the minimum Ruby version is now 2.2, and we recomment to update Rails 4.2 to a newer version?

Reply:
I think it's a good idea :)

Reply:
Yes this should mention that Rspec Rails is intended for Rails 5 and 6 and that the minimum ruby version is 2.2